src/pages/Wishlist/wishList.jsx

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useReducer, useState } from "react/cjs/react.development";
+import { useWishList } from "../../context/WishlistContext";
+import { MakiCross11 } from "../../assets/icons/icons.jsx";
+import "./wishList.css";
+import axios from "axios";
+import { useCart } from "../../context/CartContext";
+import { cartFn } from "../Cart/cartFn";
+
+export function Wishlist() {
+  const { wishList, setWishList } = useWishList();
+  const { cart, setCart } = useCart();
+  const [setBtnDisable] = useState(false);
+  const [dispatch] = useReducer(cartFn, {
+    items: [],
+    qty: 1,
+    subTotal: 0,
+  });
+  const removeHandler = (productid) => {
```

Comment:
You can create functions related to the context in the context itself. For example, if I have a wishlist context, I can define the function to add to wishlist inside the context and then I just have to call it wherever I need it. This makes the code reusable.